commons-civil/src/main/java/uk/gov/hmcts/reform/civil/bankholidays/BankHolidays.java

```diff
@@ -24,15 +24,15 @@ private Countries() {
     }
 
     @JsonIgnoreProperties(ignoreUnknown = true)
-    static class Division {
+    public static class Division {
         @JsonProperty("events")
-        List<EventDate> events;
+        public List<EventDate> events;
```

Comment:
I would double-check if this is necessary once this is merged. It shouldn't.

Reply:
yes this is as it was being used within the same package before.